app/scripts/modules/core/src/function/FunctionDetails.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Overridable, IOverridableProps } from 'core/overrideRegistry';
+
+export interface IFunctionDetailsProps extends IOverridableProps {}
+
+@Overridable('function.details')
+export class FunctionDetails extends React.Component<IFunctionDetailsProps> {
```

Comment:
We could also make this a functional component since it is presentational one-liner